modules/sndfile/sndfile.c

```diff
@@ -72,6 +83,11 @@ static SNDFILE *openfile(const struct aufilt_prm *prm, bool enc)
 			  "dump-%H-%s.wav",
 			  timestamp_print, tm, enc ? "enc" : "dec");
 
+  pl_strcpy(&file_path, filepath, 256);
+	strcat(filepath, "/");
+	strcat(filepath, filename);
+	strcpy(filename, filepath);
+
```

Comment:
I think it is better to modify the code that creates the path instead, and always insert
the path in the beginning. for example:

```
	r = re_snprintf(filename, sizeof(filename),
			  "%s/dump-%H-%s.wav",
			  path,
			  timestamp_print, tm, enc ? "enc" : "dec");
	if (r <= 0) return NULL;
```